includes/Modules/Idea_Hub.php

```diff
@@ -77,7 +101,9 @@ final class Idea_Hub extends Module
 	 * @since 1.32.0
 	 */
 	public function register() {
-		$post_meta = new Post_Meta();
+		$post_meta             = new Post_Meta();
+		$this->dismissed_items = new Dismissed_Items( $this->user_options );
+		$this->transients      = new Transients( $this->context );
```

Comment:
Looks like these instances are specific only for the `googlesitekit_admin_notices` method. In this case, let's move them into the method itself and make them be local variables for the method.

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/918df4e4ad848360545de51b12ac4167db594c2b